vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -65,7 +60,7 @@ func NewVolumeDriver(port int, useMockEsx bool, mountDir string, driverName stri
 	if useMockEsx {
 		d = &VolumeDriver{
 			useMockEsx: true,
-			ops:        vmdkops.VmdkOps{Cmd: vmdkops.MockVmdkCmd{}},
+			ops:        vmdkops.VmdkOps{Cmd: vmdkops.NewMockCmd()},
```

Comment:
`NewMockCmd` is a function that returns a new instance of `MockVmdkCmd` here. By `New` I mean a new instance here. Do you want the method name changed?